main.go

```diff
@@ -84,63 +86,62 @@ func main() {
 }
 
 func run(cfg Config, additionalSchemaPaths []string, updateDependencies bool) error {
-	return foreachChart(cfg.ChartsDirectory.path, func(base string) error {
-		logger := log.With().Str("chart", filepath.Base(base)).Logger()
-		valuesFiles, err := os.ReadDir(filepath.Join(base, TestsPath))
+	var validations_errors []error
```

Comment:
Could we use `camelCase` for the new variable names too, like `validationsErrors`?